File: Php/fibonacci.php
<?php

/**
 * Generate the Fibonacci sequence up to a given limit.
 *
 * @param int $limit The maximum number of elements in the Fibonacci sequence.
 * @return array The Fibonacci sequence up to the given limit.
 */
function fibo(int $limit)
{
    $list = array(0, 1);

    for ($i = 2; $i <= $limit; $i++) {
        $list[] = $list[$i - 1] + $list[$i - 2];
    }
    return $list;
}


$limit = 10;
var_dump(fibo($limit));

File: Python/fibonacci.py
def fibonacci(limit: int):
    """
    Generate a list of Fibonacci numbers up to the given limit.

    Parameters:
        limit (int): The maximum number of Fibonacci numbers to generate.

    Returns:
        list: A list of Fibonacci numbers.
    """
    if limit <= 0:
        return []
    elif limit == 1:
        return [0]
    elif limit == 2:
        return [0, 1]

    fib_sequence = [0, 1]
    while len(fib_sequence) < limit:
        fib_sequence.append(fib_sequence[-1] + fib_sequence[-2])

    return fib_sequence


def main():
    limit = int(input("Enter the limit of the Fibonacci sequence: "))
    fib_sequence = fibonacci(limit)
    print(f"The Fibonacci sequence up to {limit} terms is: {fib_sequence}")


if __name__ == "__main__":
    main()
